NEWS

```diff
@@ -0,0 +1,12 @@
+CHANGES IN VERSION 1.1.12
+-------------------------
+
+NEW FEATURES
+
+    o The function longestORFs, find longest ORF per
```

Comment:
I think we should not have function longestORFs as this will create a comotion and can be really easly be done with data.table basics. Remove it!